src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWAS4yv3CSWYuT4mbB7e###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_21@9
The return value of "length" must be used.

Reply:
===RESOLVED===
This issue has been resolved by a user in SonarQube